scripts/install/readarr.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+# readarr installer
+# Bakerboy448 for swizzin 2020
+# Based on Radarr v3 installationn by flying sausages
+
+#shellcheck source=sources/functions/utils
+. /etc/swizzin/sources/functions/utils
+
+_install_readarr() {
+    apt_install curl sqlite3
+
+    readarrConfDir="/home/$readarrOwner/.config/readarr"
+    mkdir -p "$readarrConfDir"
+    chown -R "$readarrOwner":"$readarrOwner" /home/$readarrOwner/.config
+
+    echo_progress_start "Downloading source files"
+    if ! curl "https://readarr.servarr.com/v1/update/nightly/updatefile?os=linux&runtime=netcore&arch=x64" -L -o /tmp/readarr.tar.gz >> "$log" 2>&1; then
+        echo_error "Download failed, exiting"
+        exit 1
+    fi
+    echo_progress_done "Source downloaded"
+
+    echo_progress_start "Extracting archive"
+    tar -xvf /tmp/readarr.tar.gz -C /opt >> "$log" 2>&1
+    echo_progress_done "Archive extracted"
+
+    touch /install/.readarr.lock
+
+    echo_progress_start "Installing Systemd service"
+    cat > /etc/systemd/system/readarr.service << EOF
+[Unit]
+Description=readarr Daemon
+After=syslog.target network.target
+
+[Service]
+# Change the user and group variables here.
+User=${readarrOwner}
+Group=${readarrOwner}
+
+Type=simple
+
+# Change the path to readarr or mono here if it is in a different location for you.
+ExecStart=/opt/readarr/readarr -nobrowser
```

Comment:
iirc wasn't it advised to add the config dir flag here? Any chance we can pass it as an environment value through the systemd script? That way anyone else can override that easily later on

Reply:
we should do that with the other arrs as well